plugins/unicodelib/unicodelib.cpp

```diff
@@ -747,6 +748,19 @@ void normalizationFormCheck(UnicodeString & source, const char * form)
     }
 }
 
+bool endsWith(UnicodeString const & processed, UnicodeString const & suffix)
+{
+    if (processed.isEmpty() || suffix.isEmpty())
+    {
+        return false;
+    }
+    if (processed.endsWith(suffix) != 1)
```

Comment:
comparison with 1 is not valid.  Should be

    if (!processed.endsWith(suffix))